### ISSUE TITLE
Too many levels of symbolic links?

### ISSUE THREAD

**Issue by snazzyjazzykenneth** [dnschneid/crouton#4420]
output of sudo edit-chroot -all
`No chroots found in /mnt/stateful_partition/crouton/chroots`

 - I haven't been able to install a chroot on an USB drive. I'm not going to pretend that I know why, I learned most of this from the unofficial/official tutorials and youtube, but my goal was to be able to run crouton on a USB drive so it didn't take up a ton of storage. I formatted the USB drive using this: https://github.com/dnschneid/crouton/wiki/Format-external-storage
 I thought I got the drive correctly formatted, but when I used the command from this tutorial to install the chroot, (https://github.com/dnschneid/crouton/wiki/How-To-Install-To-External-Drive)
It would run through the chroot, but wouldn't end up installing.
These are the first lines:

```
Installing xenial-amd64 chroot to /media/removable/San32GBext4/chroots/xenial
Downloading bootstrap files...
W: Cannot check Release signature; keyring file not available /usr/share/keyrings/ubuntu-archive-keyring.gpg
I: Retrieving InRelease 

```
Eventually in the end, I got the message:

`cp: failed to access '/media/removable/San32GBext4/bin/delete-chroot': Too many levels of symbolic links`

and the installation stopped. Is there any better way, or solution to this problem to get a chroot on an USB drive?
My chromebook model is a Lenovo 500e, Intel Celeron N3450 processor, and the USB drive is a 32 GB SanDisk Cruzer Fit USB Flash Drive.

**Comment by DennisLfromGA** [comment]
Just remount it with the 'symfollow' option.
See this comment for more details: https://github.com/dnschneid/crouton/issues/4360#issuecomment-739508057

*Hope this helps,*
-**DennisLfromGA**

**Comment by snazzyjazzykenneth** [comment]
Thanks, that worked, but when I try to start the chroot with the commands
`sudo sh startxfce4`
it gives the error:
`sh: 0: Refusing to exec startxfce4 from noexec mount; see https://chromium.googlesource.com/chromiumos/docs/+/master/security/noexec_shell_scripts.md`
Any idea what I do next?

**Comment by supechicken** [comment]
> Thanks, that worked, but when I try to start the chroot with the commands
> `sudo sh startxfce4`
> it gives the error:
> `sh: 0: Refusing to exec startxfce4 from noexec mount; see https://chromium.googlesource.com/chromiumos/docs/+/master/security/noexec_shell_scripts.md`
> Any idea what I do next?

Remount it with `exec` option should get rid of this error

**Comment by dbaynard** [comment]
I ran into this issue when attempting to enter a chroot after upgrading to chrome OS 90. I fixed it by first remounting with `symfollow` all the `/home/*` directories from `/etc/mtab/` that were mounted `nosymfollow`.

```
awk '$2 ~ /^\/home/ && $4 ~ /nosymfollow/ { print $2; }' ORS='\0' /etc/mtab | xargs -0 -I{} mount -o remount,symfollow {}
```

(I ran `mount` with sudo. Also I didn't run with the NUL byte record separator but that's because I knew my input had no spaces.)

Afterwards, enter-chroot worked correctly.
